zilliqa/src/exec.rs

```diff
@@ -1070,6 +1071,18 @@ impl PendingState {
         }
     }
 
+    pub fn get_zil_chain_id(&self) -> u64 {
```

Comment:
Tiny nit but I prefer 
```suggestion
    pub fn zil_chain_id(&self) -> u64 {
```
:)